### DISCUSSION TITLE
npm-how-to-document-your-package-json-scripts

### DISCUSSION THREAD

**Discussion by giscus** [strdr4605/strdr4605.github.io#71] [General]
# npm-how-to-document-your-package-json-scripts

Let's see different ways you can document your npm scripts using README, comments in package.json, or using npx why package.

https://strdr4605.com/npm-how-to-document-your-package-json-scripts

**Comment by zhangfisher** [comment]
using the Vscode plugin `JsonComments`. It's a handy plugin that allows you to add comments to any JSON file.

[JsonComments](https://marketplace.visualstudio.com/items?itemName=wxzhang.json-comments&ssr=false#review-details)
